docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -2,17 +2,20 @@
 
 = Comparing configuration files between deployments
 
-In order to help users to handle the configuration for the {OpenStackPreviousInstaller} and {rhos_prev_long}
-services the tool: https://github.com/openstack-k8s-operators/os-diff has been
-develop to compare the configuration files between the {OpenStackPreviousInstaller} deployment and the {rhos_long} cloud.
-Make sure Golang is installed and configured on your environment:
+To help you manage the configuration for your {OpenStackPreviousInstaller} and {rhos_prev_long} ({OpenStackShort}) services, you can compare the configuration files between your {OpenStackPreviousInstaller} deployment and the {rhos_long} cloud by using the os-diff tool.
 
+.Prerequisites
+
+* Golang is installed and configured on your environment:
++
 ----
 dnf install -y golang-github-openstack-k8s-operators-os-diff
 ----
 
-Then configure the `/etc/os-diff/os-diff.cfg` file and the `/etc/os-diff/ssh.config` file according to your environment. To allow os-diff to connect to your clouds and pull files from the services that you describe in the `config.yaml` file you need to properly set the option in the `os-diff.cfg` file:
+.Procedure
 
+. Configure the `/etc/os-diff/os-diff.cfg` file and the `/etc/os-diff/ssh.config` file according to your environment. To allow os-diff to connect to your clouds and pull files from the services that you describe in the `config.yaml` file, you must set the option in the `os-diff.cfg` file:
```

Comment:
What does "the option" refer to? `ssh_cmd`? `director_host`? Both, if applicable?